clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesApiResourceParser.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.security;
+
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesApiGroup;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesKind;
+import java.util.HashSet;
+import java.util.Set;
+
+public class KubernetesApiResourceParser {
+
+  public static Set<KubernetesKind.ScopedKind> parse(String input) {
+    String[] lines = input.trim().split("\n");
+    String headerRow = lines[0];
+    int apiGroupIndex = headerRow.indexOf("APIGROUP");
+    int namespaceIndex = headerRow.indexOf("NAMESPACED");
+    int kindIndex = headerRow.indexOf("KIND");
+
+    Set<KubernetesKind.ScopedKind> kinds = new HashSet<>();
```

Comment:
It might be good to throw an explicit exception here if the data is not of the format we expect (ie, if we don't find those headers).  Otherwise it will probably be a reasonably hard-to-debug NPE or out-of-bounds exception that happens downstream.